cwf/gateway/fabfile.py

```diff
@@ -199,12 +199,12 @@ def integ_test(
     # HSSLESS tests are to be executed from gateway_host VM
     if tests_to_run.value == SubTests.HSSLESS.value:
         _run_integ_tests(
-            c, gateway_host, trf_host, tests_to_run, test_re, count,
+            gateway_host, trf_host, tests_to_run, test_re, count,
             test_result_xml, rerun_fails, c_cwf, c_trf,
         )
     else:
         _run_integ_tests(
-            c, test_host, trf_host, tests_to_run, test_re, count,
```

Comment:
This should be tested. I'm not sure if the connection needs to be parsed even if it is not explicitly called in the function and gets flagged as unused IDE.